### DISCUSSION TITLE
Reliable way to play a single file after every track

### DISCUSSION THREAD

**Discussion by neownd** [AzuraCast/AzuraCast#5318] [General]
Hello,
I'm trying to get the following work:
- I have a simple playlist which plays songs randomly
- After each song, Azuracast/LiquidSoap must play a specific file
- Or easier explained: Always play a jingle after each other element (Song, Jingle, Song, Jingle, Song, ...)

At the moment I have the following settings:
- Music playlist with priority 1 and lots of files
- Jingle playlist with priority 1  and just the one, single file
- Jingle playlist is set to "Play every 1 song"
- "Jingle Mode" is turned off, so jingle should appear in the song history

What happens (and what happens not):
- Jingle gets not played every 1 song, but every 1-3 songs in average (it's not as reliable as it should be)
- Jingle does not always appear in the song history

The "jingle" is just a 5 second long file which is almost silent (actually it's not a real jingle but a silent file which contains a 1 second long, 18 Hz tone to trigger audio ads). Is the short length and the silence maybe a problem which makes LS skip the file sometimes? (No, the listener won't hear ads after each song, it's a smart adserver system).

I tried several other things like playing around with priorities, setting the jingle playlist to "play every 1 minute" etc. I also thought about adding a custom line to LS config to do the job, but unfortunately my skills are not good enough. In fact, I want to keep it as simple as possible since I have to roll our the settings, once they work, to a huge number of channels.

**Comment by cybercuate** [comment]
same

**Comment by freqq68** [comment]
same too,that be cool if we can have a jingle cat. 

That can set, like playing this jingle at hour XX or a Rotation like module ^^

In radioboss, I can call 1 jingle for a specific hour like "it's 3 on the morning" >> after we get the normal rotation :)
